src/index.html

```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
```

Comment:
If we have no content, we can use a single tag, but this is optional
```suggestion
      <div class="stars__star" />
```